server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/LoadFiles.java

```diff
@@ -142,7 +141,7 @@ public Repo<Manager> call(final long tid, final Manager manager) {
   }
 
   // visible for testing
-  public abstract static class Loader {
+  public abstract static class Loader implements AutoCloseable {
```

Comment:
The reason AutoCloseable was added here was to support shutting down the thread pool using try w/ resources. Did this to avoid leaking threads when exceptions happen.